pkg/js/transformer/modules/modules-to-model.ts

```diff
@@ -0,0 +1,191 @@
+import { AuthorizationModel, Condition , TypeDefinition } from "@openfga/sdk";
+import { validateJSON } from "../../validator";
+import { transformModularDSLToJSONObject } from "../dsltojson";
+import {
+  BaseError,
+  DSLSyntaxError,
+  ModelValidationError,
+  ModuleTransformationError,
+  ModuleTransformationSingleError
+} from "../../errors";
+import { getTypeLineNumber, getConditionLineNumber, getRelationLineNumber } from "../../util/line-numbers";
+import { constructTransformationError } from "../../util/exceptions";
+
+export interface ModuleFile {
+  name: string;
+  contents: string;
+}
+
+export const transformModuleFilesToModel = (files: ModuleFile[]): Omit<AuthorizationModel, "id"> => {
```

Comment:
Question, should this function also require the fga.mod json? FOr the future so we're not assuming 1.2

Reply:
Yeah I was meaning to add that and forgot.

 I wasn't sure whether to take the whole FGAMod interface or have a second optional parameter that makes it easier to add extras onto in future (like if we ever wanted to allow specifying `ValidationOptions` for the `validateJSON` call)

Reply:
#188 188